### PR TITLE
chore(console): improve text

### PR DESCRIPTION
**Pull request by fforootd**

### CODE REVIEW

File: console/src/assets/i18n/de.json [hunk 1]
```diff
@@ -128,102 +127,103 @@
...
         },
         "SEARCH":{
-            "FOUND":"gefunden"
+            "FOUND":"Gefunden"
         },
         "TARGET": {
-            "SELF":"Suchen Sie oben nach Usern ihrer Organisation, Zum hinzufügen externer Benutzer",
-            "EXTERNAL":"Um User Ihrer Organisation hinzuzufügen",
+            "SELF":"Suchen Sie nach den Benutzern ihrer Organisation, Zum hinzufügen externer Benutzer", #DAFUQ
+            "EXTERNAL":"Um User Ihrer Organisation hinzuzufügen", #DAFUQ
```
Comment (fforootd) [inline console/src/assets/i18n/de.json:188]: @fgerschwiler @peintnermax i have no clue what this does 😁

File: console/src/assets/i18n/en.json [hunk 2]
```diff
@@ -154,54 +153,54 @@
...
         },
         "SEARCH":{
-            "FOUND":"found"
+            "FOUND":"Found"
         },
         "TARGET": {
-            "SELF":"You will get users of your organization. If you want to add an external user",
-            "EXTERNAL":"To get users of your organization",
+            "SELF":"You will get users of your organisation. If you want to add an external user", #DAFUQ
+            "EXTERNAL":"To get users of your organisation", #DAFUQ
```
Comment (fforootd) [inline console/src/assets/i18n/en.json:188]: @fgerschwiler @peintnermax i have no clue what this does 😁

Comment (fgerschwiler) [reply console/src/assets/i18n/en.json:188]: if youd like to grant an user of an other organisation you need to click the link, because its another search

Comment (fgerschwiler) [reply console/src/assets/i18n/en.json:188]:
```suggestion
            "SELF":"If you want to grant a user of an other organisation",
            "EXTERNAL":"To grant a user of your organisation",
```

File: console/src/assets/i18n/de.json (hunk 1 above)
Comment (fgerschwiler) [reply console/src/assets/i18n/de.json:188]:
```suggestion
            "SELF":"Um einen Benutzer aus einer anderen Organisation zu berechtigen",
            "EXTERNAL": "Um einen Benutzer ihrer eigenen Organisation zu berechtigen", 
``

**Pull request merged**